scripts/utils/trading_strategy_helpers.js

```diff
@@ -75,6 +77,33 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
     return orders != null
   }
 
+  /**
+   * Returns the symbol of the ERC20 token in input.
+   * Returns the value from a list if present, otherwise executes a contract call.
+   *
+   * @param {string} method the name of the methods that (can be "symbol", "decimals", or "name")
+   * @param {SmartContract} tokenInstance instance of the token at the previous address
+   * @returns {string|number} the result of calling the selected method
+   */
+  const tokenDetail = async function (method, tokenInstance) {
+    let detail
+    const tokenAddress = tokenInstance.address
+    const networkId = tokenInstance.constructor.network_id
+    if (hardcodedTokensByNetwork[networkId] && hardcodedTokensByNetwork[networkId][tokenAddress]) {
+      return hardcodedTokensByNetwork[networkId][tokenAddress][method]
+    }
```

Comment:
this project also contains a few other hard coded tokens by network:

https://github.com/gnosis/dex-liquidity-provision/blob/02e942f65b1a72671d2da455ab0fa2e9c655e9d1/scripts/synthetix/facilitate_trade.js#L23-L47

Not sure if it might be a good idea to have all these items in the same place. It is beginning to feel like it seems the synthetix scripts are entirely independent from the rest of this project and should perhaps be moved into their own repo.